boat_nav/src/navigator_node.py

```diff
@@ -497,12 +447,14 @@ def taras_algorithm():
 		else:
 			boat_dir = -1 
 		
-		wind_coming = (apparent_wind_heading + 180) % 360 # Which direction the wind is coming from
+		wind_coming = angles.normalize(apparent_wind_heading + 180) # Which direction the wind is coming from
 		
-		if (boat_dir is 1 and not is_within_bounds(wind_coming, cur_boat_heading, target_heading)) or\
-			(boat_dir is -1 and is_within_bounds(wind_coming, cur_boat_heading, target_heading)):
+		# TODO: This may be broken now that is_within_bounds is not bi-directional
```

Comment:
Why is is_within_bounds not bidirectional? Have you tested this line?

Reply:
See https://github.com/uwsailbot/boat_2018/pull/21#discussion_r236469543.

I haven't tested this code, but looking at the rest of it it doesn't look like it works anyways. It was never really functional. I'm somewhat inclined to just remove it rather than trying to maintain old, broken code.